helsingborg-stad/app-mitt-helsingborg
# Bug/truncated calculations

## Explain the changes you’ve made
Fixed a styling issue where the different values where not shown correctly

## Explain why these changes are made
Things should not look bad :D 

## Explain your solution
Fixed the width of the first column containing name etc. Also remove an "invisible" column which were adding for spacing.
Added properties to give the values more space and width

## How to test

Concrete example:

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Make sure you have a closed case with calculations

## Tested environments

- [] Storybook on a iOS device/simulator.

## Screenshots
**Before**
![image](https://user-images.githubusercontent.com/9610681/176422354-66f27c06-5d04-4f40-a7a5-d8b71592db00.png)

**After**
![simulator_screenshot_94E19E89-0D45-494C-B43E-9A3BAD51F03E](https://user-images.githubusercontent.com/9610681/176423378-949f609d-1a43-40b1-be14-332de8dded7d.png)
